src/main/java/club/sk1er/patcher/config/PatcherConfig.java

```diff
@@ -564,6 +564,13 @@ public static void modifyEverySound() {
         GuiUtil.open(Objects.requireNonNull(Patcher.instance.getPatcherSoundConfig().gui()));
     }
 
+    @Property(
+        type = PropertyType.SWITCH, name = "Natural Capes",
+        description = "Render capes more naturally",
```

Comment:
Be more descriptive, don't just restate the feature name.

Reply:
Not great at English isn't my greatest but will do.